jitsi-utils/src/main/java/org/jitsi/utils/queue/AsyncQueueHandler.java

```diff
@@ -45,16 +45,23 @@
      * The default <tt>ExecutorService</tt> to run <tt>AsyncQueueHandler</tt>
      * when there is no executor injected in constructor.
      */
-    private final static ExecutorService sharedExecutor
-        = Executors.newCachedThreadPool(
-            new CustomizableThreadFactory(
-                AsyncQueueHandler.class.getName() + "-executor-", true));
+    private final static Executor sharedExecutor = new Executor() {
```

Comment:
So this is not using the safe variant?

Reply:
No, it uses the `Future`. It does it's own exception handling, at [least for the `handleItem` method](https://github.com/jitsi/jitsi-utils/blob/master/jitsi-utils/src/main/java/org/jitsi/utils/queue/AsyncQueueHandler.java#L139)